### PR TITLE
feat: Nyaa blue alt orange and seadex links

### PR DESCRIPTION
**Pull request by Kapppa**
- Makes alt releases orange
- Adds a SeaDex button to the page so you can go directly to SeaDex page.
- Increase limit to 75 so whole page can be seen. (Nyaa page is 75 results)

### CODE REVIEW

File: sk/static/nyaablue.user.js [hunk 1]
```diff
@@ -39,22 +40,49 @@ async function setViewBlue () {
 
   const { items } = await torrentsResponse.json()
   if (items[0]?.infoHash) {
-    element?.classList.add('panel-info')
+    element?.classList.add(items[0]?.isBest ? 'panel-info' : 'panel-info-alt')
+  }
+  if (items[0]?.id) {
+    const collectionResponse = await seadexEntryEndpoint('', {
+      filter: 'trs~"' + items[0]?.id + '"'
```
Comment (ThaUnknown) [inline sk/static/nyaablue.user.js]: why.... not infoHash search? %LIKE% is fucking slow

Comment (Kapppa) [reply sk/static/nyaablue.user.js]: This isn't torrents. This is entries.

File: sk/static/nyaablue.user.js [hunk 2]
```diff
@@ -39,22 +40,49 @@ async function setViewBlue () {
...
+
+    const entries = await collectionResponse.json()
+
+    if (entries.items?.length !== 0) {
+      const report = document.querySelector('body > div > div:nth-child(1) > div.panel-footer.clearfix > button')
+
+      document.head.insertAdjacentHTML('beforeend', '<style id="css_blue" type="text/css">button.btn-seadex {margin-right: 5px; color: #fff; background-color: #247fcc; border-color: #247fcc;} button.btn-seadex:hover {margin-right: 5px; color: #fff; background-color: #19578b; border-color: #19578b;} </style>')
+
+      for (const [i, info] of entries.items?.entries()) {
+        report?.insertAdjacentHTML("afterend", `<button type="button" id="md-seadex-${i}" class="btn btn-xs btn-seadex pull-right">Seadex</button>`)
```
Comment (ThaUnknown) [inline sk/static/nyaablue.user.js]: no....
```js
const button = document.createElement('button')
button.classList... smth i dont remember
button.onclick = () => {}
report.append(button)
```

File: sk/static/nyaablue.user.js [hunk 3]
```diff
@@ -39,22 +40,49 @@ async function setViewBlue () {
...
+    const collectionResponse = await seadexEntryEndpoint('', {
+      filter: 'trs~"' + items[0]?.id + '"'
+    })
+
+    const entries = await collectionResponse.json()
+
+    if (entries.items?.length !== 0) {
+      const report = document.querySelector('body > div > div:nth-child(1) > div.panel-footer.clearfix > button')
+
+      document.head.insertAdjacentHTML('beforeend', '<style id="css_blue" type="text/css">button.btn-seadex {margin-right: 5px; color: #fff; background-color: #247fcc; border-color: #247fcc;} button.btn-seadex:hover {margin-right: 5px; color: #fff; background-color: #19578b; border-color: #19578b;} </style>')
```
Comment (ThaUnknown) [inline sk/static/nyaablue.user.js]: shouldnt that go [here?](https://github.com/ThaUnknown/releases-moe/pull/40/files#diff-ec9a791432ddc2e19aa339916a454f4b1e50cb674ec83e888a27c81f66343673R102)

Comment (Kapppa) [reply sk/static/nyaablue.user.js]: No point in adding it if it doesn't need to.

File: sk/static/nyaablue.user.js (hunk 1 above)
Comment (ThaUnknown) [reply sk/static/nyaablue.user.js]: ...and? you can search by relation

File: sk/static/nyaablue.user.js [hunk 4]
```diff
@@ -39,22 +40,49 @@ async function setViewBlue () {
...
+    element?.classList.add(items[0]?.isBest ? 'panel-info' : 'panel-info-alt')
+  }
+  if (items[0]?.id) {
+    const collectionResponse = await seadexEntryEndpoint('', {
+      filter: 'trs~"' + items[0]?.id + '"'
+    })
+
+    const entries = await collectionResponse.json()
+
+    if (entries.items?.length !== 0) {
```
Comment (ThaUnknown) [inline sk/static/nyaablue.user.js]: if items is undefined this will run, remove the 0 check

**Review by ThaUnknown — APPROVED** [review]

**Comment by ThaUnknown** [comment]
ya will need to test if it works urself, but code looks good

**Pull request merged**